src/components/ProjectList/listOfProjects.js

```diff
@@ -42,6 +42,12 @@ const projectList = [
     projectLink: 'https://github.com/exercism/exercism/contribute',
     description: 'Quickly ramp up in new programming languages!',
     tags: ['Ruby', 'Exercises', 'CLI', 'Web App'],
+  }, {
+    name: 'React Testing Library',
+    imageSrc: 'https://raw.githubusercontent.com/testing-library/react-testing-library/master/other/goat.png',
+    projectLink: 'https://github.com/testing-library/react-testing-library',
```

Comment:
Hey @gabrielsanttana can we point the link to issue page or default <projectname>/contribute page as generated by github?

Reply:
Yeay, sure!